module/documents/actor.mjs

```diff
@@ -64,6 +64,17 @@ export default class ActorEd extends Actor {
     return this.items.filter( item => item.type === "discipline" );
   }
 
+  /**
+   * Returns the highest discipline of an actor
```

Comment:
That's great! I'd try to slightly improve the doc: I think type is correct, but then you need to adjust the initial value in the reduce to `undefined` instead of `null`. Which is semantically better anyway.
Also, I would be more specific and say that we return the discipline with the highest circle, and if there are multiple with the same, it's the first encountered. No order guaranteed.

Reply:
ah right, of course that makes sense